src/CareTogether.Api/OData/LiveODataModelController.cs

```diff
@@ -25,6 +26,8 @@ namespace CareTogether.Api.OData
 {
     public sealed record Location([property: Key] Guid Id, Guid OrganizationId, string Name);
 
+    public sealed record LocationUsers([property: Key] Guid LocationId, Guid[] UserIds);
```

Comment:
Actually better to make this flat, with a _compound key_:
```suggestion
    public sealed record LocationUserAccess([property: Key] Guid UserId, [property: Key] Guid LocationId);
```

Reply:
The "Access" suffix helps suggest that in the future we might add `Role` or related info.